db/migrate/20230503144046_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
```

Comment:
- [ ] Same here about references, `t.references :author, foreign_key: { to_table: 'users' }` and `t.references :post, foreign_key: true`. This will create `author_id`, `post_id`, and a corresponding index in the comments table 🙃 🙃

Reply:
- [ ] Kindly address this part of the comment as you did to the likes table